### PR TITLE
Scrollbar for menu options

### LINKED ISSUE #508: Do not shrink menu options when the editor width is constrainted
Ref: https://vimeo.com/785142397/76009d36f2

The menu options must always take up the full size. If the size is constrained, show a scrollbar.

@gaagul _a

### PR DESCRIPTION
**Pull request by gaagul**
Fixes #508

Description

- Changed: Updated editor menu style to enable scroll when editor width is constrained.

Reviewers

@AbhayVAshokan _a

### CODE REVIEW

**Comment by AbhayVAshokan** [comment]
@gaagul _a assign the PR back to me after making the following changes.

1. Update the PR description as we discussed earlier. You can refer to this PR for reference: https://github.com/bigbinary/neeto-editor/pull/456
2. As I mentioned earlier, you need to add a proper label to the PR as well. Since this is a minor UI change, you need to add a `patch` label.

**Comment by gaagul** [comment]
@AbhayVAshokan _a

File: lib/styles/editor/_menu.scss [hunk 1]
```diff
@@ -16,17 +16,19 @@
...
     }
   }
 
+  
   &__wrapper {
     display: flex;
     justify-content: flex-start;
     align-items: center;
-    overflow: auto;
+    overflow: scroll;
```
Comment (AbhayVAshokan) [inline lib/styles/editor/_menu.scss:31]:
```suggestion
    overflow: auto;
```

This change is not necessary.
When `overflow` is scroll: The scrollbar is always shown (even if the content does not overflow).
When `overflow` is auto: The scrollbar is shown only when the content overflows the container. Hence, better UX.

Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/overflow

File: lib/styles/editor/_menu.scss [hunk 2]
```diff
@@ -16,17 +16,19 @@
   }
 
   button.neeto-editor-fixed-menu__item {
+    min-width: fit-content;
```
Comment (AbhayVAshokan) [inline lib/styles/editor/_menu.scss:19]: Add a blank line before the next class.

**Review by AbhayVAshokan — APPROVED** [review]
@gaagul _a

**Comment by gaagul** [comment]
@AbhayVAshokan _a I have made the changes, Please review.

**Review by AbhayVAshokan — APPROVED** [review]

**Pull request merged**